### ISSUE TITLE
Request: adding `CITATION.cff` and updating citation wording

### ISSUE THREAD

**Issue by jamesbraza** [nipy/nibabel#1230]
https://github.com/nipy/nibabel#citation talks about a DOI badge:

> NiBabel releases have a [Zenodo](https://zenodo.org/) [Digital Object Identifier](https://en.wikipedia.org/wiki/Digital_object_identifier) (DOI) badge at the top of the release notes

The first request is can the wording be updated to say "top of the `README.md`", as there doesn't seem to be release notes.

---

The other request is can we add a `CITATION.cff` file to the repo root, to make it easy to cite in academic papers?

You can read about [`CITATION.cff` here](https://citation-file-format.github.io/) and use https://bit.ly/cffinit to easily generate a `CITATION.cff` file.

Simply adding it to the repo root adds this menu to the right panel:

![image](https://github.com/nipy/nibabel/assets/8990777/58f451f3-9396-48af-bd97-2a66c2161330)

**Comment by effigies** [comment]
The release notes are in https://github.com/nipy/nibabel/releases.

I'm not sure what's difficult about citing using Zenodo releases, but I'd be okay with adding a `CITATION.cff`, if it's useful. From a quick glance at those docs, it looks like you have to hard-code the version and DOI, which is problematic. We use git tags to determine the version, and adding updating that to the release process would add a potential source of error. Further, the version-specific DOI from Zenodo is unknown until after the release is published on GitHub.

Anyway, if you have an example of a repository using them that still allows versions to be single-sourced from git tags, I'd be willing to consider this.

**Comment by jamesbraza** [comment]
Oh I see what you mean by release notes now, I was looking for a file in the repo.

Yeah I think I requested `CITATION.cff` because I was having a hard time finding the Zenodo link.  I made https://github.com/nipy/nibabel/pull/1231 to address this for future users.

I don't know how to integrate `CITATION.cff` with Git tags, and couldn't find anything about it in their GitHub issues, so I posted a question there.

Feel free to close this out! 👍
